padertorch/data/fragmenter.py

```diff
@@ -107,3 +107,95 @@ def fragment(key, x):
             fragment = deflatten(fragment)
             fragments.append(fragment)
         return fragments
+
+
+def get_segment_offsets(num_samples, segment_length, step=None,
+                        offset_mode='start', num_segments=None):
+    """
+
+    Args:
+        num_samples:
+        segment_length:
+        shift:
+        offset_mode:
+        num_segments:
+
+    Returns:
+    >>> get_segment_offsets(25, 10, None, offset_mode='start')
+    array([ 0, 10])
+    >>> get_segment_offsets(25, 10, 3, offset_mode='start')
+    array([ 0,  3,  6,  9, 12, 15])
+    >>> get_segment_offsets(24, 10, 3, offset_mode='start')
+    array([ 0,  3,  6,  9, 12])
+    >>> get_segment_offsets(24, 10, 3, offset_mode='end')
+    array([ 2,  5,  8, 11, 14])
+    >>> get_segment_offsets(24, 10, 3, offset_mode='center')
+    array([ 1,  4,  7, 10, 13])
+    >>> get_segment_offsets(24, 10, 3, offset_mode='center', num_segments=2)
```

Comment:
What is the advantage of using `num_segments` compared to `get_segment_offsets(...)[:num_segments]`?

Reply:
At the moment there is none. I was thinking about making it random. However, I was not sure whether anybody was interested in that. I am not using this num_segments feature so I am not sure what the expectations are.